### PR TITLE
Update markdown syntax

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #3)

### CODE REVIEW

File: doc/content/manuals/include/fault_rupture/fault_rupture-theory.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ M_{zz}(t) &= M_o(t) \sin 2\delta \sin \lambda
 
 Each component of the above matrix is a force couple with the first index representing the force
 direction and the second index representing the direction in which the forces are separated (see
-[fig:source_direction]; [citet:aki2012quantitative]).
+[fig:source_direction]; [!citet](aki2012quantitative)).
```
Comment (cbolisetti) [inline doc/content/manuals/include/fault_rupture/fault_rupture-theory.md]: Its not working here for some reason..

File: doc/content/manuals/include/materials/isoil-user.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 The I-soil material model is a nonlinear hysteretic soil model that is based on the distributed
-element models developed by [citet:iwan1967on] and [citet:chiang1994anew]. In 1-D, this model takes
+element models developed by [!citet](iwan1967on) and [!citet](chiang1994anew). In 1-D, this model takes
 the backbone stress-strain curve and divides it into a set of elastic-perfectly plastic curves. The
 total stress then is the sum of the stresses from the individual elastic-perfectly plastic curves.
 
 The three dimensional generalization of this model is achieved using von-Mises failure criteria for
 each elastic-perfectly plastic curve resulting in an invariant yield surfaces in three-dimensional
-stress space like in [fig:yieldsurface] (after [citet:chiang1994anew]).
+stress space like in [fig:yieldsurface] (after [!citet](chiang1994anew)).
```
Comment (cbolisetti) [inline doc/content/manuals/include/materials/isoil-user.md]: Same here.

File: doc/content/manuals/include/fault_rupture/fault_rupture-theory.md (hunk 1 above)
Comment (aeslaughter) [reply doc/content/manuals/include/fault_rupture/fault_rupture-theory.md]: You found a super tricky regex problem, I will see if I can solve it but for now just use the suggestion.

Comment (aeslaughter) [reply doc/content/manuals/include/fault_rupture/fault_rupture-theory.md]:
```suggestion
[fig:source_direction]; 
[!citet](aki2012quantitative)).
```

File: doc/content/manuals/include/materials/isoil-user.md (hunk 2 above)
Comment (aeslaughter) [reply doc/content/manuals/include/materials/isoil-user.md]:
```suggestion
stress space like in [fig:yieldsurface]
(after [!citet](chiang1994anew)).
```

Comment (cbolisetti) [reply doc/content/manuals/include/materials/isoil-user.md]: I am not able to commit those suggestions. Can you please commit? I can merge then.

**Pull request merged**